server/src/main/java/org/elasticsearch/client/support/AbstractClient.java

```diff
@@ -211,18 +205,8 @@ public CompletableFuture<AcknowledgedResponse> delete(final DeleteIndexRequest r
         }
 
         @Override
-        public ActionFuture<AcknowledgedResponse> putMapping(final PutMappingRequest request) {
-            return legacyExecute(PutMappingAction.INSTANCE, request);
-        }
-
-        @Override
-        public PutMappingRequestBuilder preparePutMapping(String... indices) {
-            return new PutMappingRequestBuilder(this, PutMappingAction.INSTANCE).setIndices(indices);
-        }
-
-        @Override
-        public ActionFuture<UpgradeResponse> upgrade(final UpgradeRequest request) {
```

Comment:
This is now executed only through `OPTIMIZE` sql stmt, correct?

Reply:
yep